webapp/first-snap/python/test.yaml

```diff
@@ -0,0 +1,14 @@
+macos:
+  - action: 'Install Multipass (https://github.com/CanonicalLtd/multipass/releases/), a virtualisation product that can set up a complete Linux environment in a couple of minutes'
```

Comment:
Could this be a link? It looks really weird on the page. I thought it was a unparsed Markdown or something... 
<img width="697" alt="screen shot 2018-10-19 at 15 42 51" src="https://user-images.githubusercontent.com/83575/47221791-ab394a80-d3b5-11e8-821f-27924b0dcf1b.png">